crypto/crypto.go

```diff
@@ -220,35 +219,47 @@ func toECDSA(d []byte, strict bool) (*ecdsa.PrivateKey, error) {
 	return priv, nil
 }
 
-// ReadPrivKey reads a private key from the file path
-func ReadPrivKey(path string) (*ecdsa.PrivateKey, error) {
-	createFn := func() ([]byte, error) {
-		key, err := GenerateKey()
-		if err != nil {
-			return nil, err
-		}
+// generateKeyAndMarshall generates a new private key and serializes it to a byte array
+func generateKeyAndMarshall() ([]byte, error) {
```

Comment:
`Marshall` is the military title, `Marshal` is a verb meaning putting something in order.

This is me giving back the favour for `shouldSuccess` :D

Reply:
Just kidding ofcourse I just know the https://pkg.go.dev/encoding/json package has Marshal methods with single `l`

Reply:
Fixed the typo for other functions as well in the `crypto.go` file 👍